config/routes.rb

```diff
@@ -14,6 +14,7 @@
   get 'export_users' => 'users#export_user'
   post 'import_users' => 'users#import_user', as: 'import_users'
   resources :students
+  get 'students/:id/history' => 'students#history', as: 'history'
```

Comment:
You can define this as route inside `resources :students` (https://guides.rubyonrails.org/routing.html#adding-member-routes)

Reply:
thanks